src/algorithms/digi/TOFHitDigiConfig.h

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Souvik Paul
+
+#pragma once
+
+#include <DD4hep/DD4hepUnits.h>
+
+namespace eicrecon {
+
+struct TOFHitDigiConfig {
+  // single hit energy deposition threshold
+  double threshold{1.0 * dd4hep::keV};
+  double tRes = 0.1; /// TODO 8 of what units??? Same TODO in juggler. Probably [ns]
+                     // digitization settings
+  double resolutionTDC{1};
+  double resolutionADC{1};
+
+  // Parameters of AC-LGAD signal generation - Added by Souvik
```

Comment:
Keeping with my point below: Souvik is accredited in the copyright here.
```suggestion
  // Parameters of AC-LGAD signal generation
```

Reply:
Comments removed